### PR TITLE
[ADD]: to-do documentation

### PR DESCRIPTION
**Pull request by Donapi**
[Task ID: 3193463](https://www.odoo.com/web#id=3193463&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/productivity/to_do.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
+
+=====
+To-do
+=====
+
+The To-Do app assists users in efficiently organizing and managing their work. It enables seamless
+collaboration and streamlined workflows across different aspects of the business, offering features
+such as creating and assigning tasks, setting deadlines, creating memos and to-do lists, monitoring
+progress, and much more.
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:9]: I would say tasks or write something more targeted to "to do" and doing tasks, can be great but this paragraph makes the app seems so great and powerful but it's only a to-do.

To-Do assists users in efficiently organizing and managing their tasks. It enables collaboration and faster workflows 
across different aspects of the business. It offers features such as creating and assigning tasks, setting deadlines, creating memos and to-do lists, monitoring progress, and much more.

File: content/applications/productivity/to_do.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Configuration
+=============
+
+If To-Do is not yet installed on your database, go to the :guilabel:`Apps`, search for it, and click
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:14]: If :guilabel:'To-Do' is not yet installed on your database ...

File: content/applications/productivity/to_do.rst [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+:guilabel:`Install`.
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:15]: I would add a link to this page, explaining how to install a module
https://www.odoo.com/documentation/16.0/applications/general/apps_modules.html#general-install

File: content/applications/productivity/to_do.rst [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+.. image:: to_do/install-to-do.png
+   :align: center
+   :alt: Install To-do app
+
+.. _to-do/personal-stages:
+
+Personal stages
+===============
+
+Your assigned to-dos are displayed on your dashboard in different by-default stages:
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:26]: Your assigned to-dos are displayed on your dashboard in different default stages:

File: content/applications/productivity/to_do.rst [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+Different actions are possible in a to-do:
+
+- :guilabel:`Mark as done`: click the check mark button when the task has been completed;
+- :guilabel:`Archive`, :guilabel:`Duplicate`, :guilabel:`Delete`, :guilabel:`Dashboard`, or
+  :guilabel:`Convert to task`: click on the cog icon to see these options;
+- :guilabel:`Tags`: select tags from the dropdown list to categorize your to-do. You can create a
+  new tag simply by writing it;
+- :guilabel:`Assignees`: select one or more users from the dropdown list to assign your to-do;
+- **Customize your tasks content**: when typing / in the text area, a dropdown list of features
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:71]: typing `/`

File: content/applications/productivity/to_do.rst [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+.. image:: to_do/task-edition.png
+      :align: center
+      :alt: edit your task
+
+Scheduling activities
+=====================
+
+To help you get things done, you can schedule activities. From a to-do, click on
+:guilabel:`Activities`; or from the dashboard, click the clock icon on a to-do card, then
+:guilabel:`+ schedule an activity`:
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:89]: You can schedule activities from a to-do, click on Activities; or from the dashboard, click the clock icon on a to-do card, then + schedule an activity:

File: content/applications/productivity/to_do.rst [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+.. image:: to_do/schedule-activity.png
+      :align: center
+      :alt: schedule an activity
+
+Then, fill in the required fields:
+
+- :guilabel:`Activity type`: select the type of activity from the list;
+- :guilabel:`Summary`: add a brief description of the activity;
+- :guilabel:`Due date`: select the date the activity is due to;
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:99]: - :guilabel:`Due date`: select the date the activity is due;

File: content/applications/productivity/to_do.rst [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+- :guilabel:`Assigned to`: select an assignee for the activity;
+- :guilabel:`Log a note`: you can add further description to the activity.
+
+Click :guilabel:`Schedule`.
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:103]: Click :guilabel:`Schedule` to complete the action.

File: content/applications/productivity/to_do.rst [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+Convert to-dos into tasks
+=========================
+
+You can convert a to-do into a task and assign it to a project by going to the to-do you want to
+convert, click the cog icon and :guilabel:`Convert to Task`. Select the :guilabel:`Project` you
+want, add :guilabel:`Assignees`, :guilabel:`tags`, and click :guilabel:`Convert to Task`. The to-do
+is now like a private task and appears in the Project app under :guilabel:`My Tasks`.
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:111]: You can convert a to-do into a task and assign it to a project. You can do so by going to the to-do you want to convert, clicking the cog icon, and :guilabel:`Convert to Task`. Select the :guilabel:`Project` you want, add :guilabel:`Assignees`, :guilabel:`tags`, and click :guilabel:`Convert to Task`. The to-do is now a private task in the Project app under :guilabel:`My Tasks`.

File: content/applications/productivity/to_do.rst [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+
+.. tip::
+   When converting a to-do into a task, add or remove assignees to control who has access to the
+   task. Note that its visibility also depends on the visibility settings of the project you are
+   assigning it to.
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:116]: Add or remove assignees to control who can access the task when converting it. Note that its visibility also depends on the visibility settings of the project you are assigning it to.

File: content/applications/productivity/to_do.rst [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+Additional features
+===================
+
+Click the cog icon to display additional features:
+
+.. image:: to_do/additional-features.png
+      :align: center
+      :alt: additional features
+
+:guilabel:`Import records`: to upload an Excel or CSV file to import in your to-do. Click
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:127]: :guilabel:`Import records`: to upload an Excel or CSV file to import into your to-do.

File: content/applications/productivity/to_do.rst [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+:guilabel:`Spreadsheet`: to insert a link to your to-dos in a spreadsheet, select your spreadsheet
+and click :guilabel:`confirm`.
+
+.. tip::
+   - Use the search bar to filter, group by, or save your search.
+   - Use the different views to display your to-dos:
+
+     - :guilabel:`Kanban`: it displays records as “cards” among different stages;
+     - :guilabel:`List`: records are listed and grouped by stage;
+     - :guilabel:`Activity`: used to display the activities linked to the to-dos. Data are displayed
```
Comment (chiaraprattico) [inline content/applications/productivity/to_do.rst:147]: Data is displayed

**Review by chiaraprattico — CHANGES_REQUESTED** [review]
Hi @Donapi, nice doc, we can check this together. This PR is targeting 16.0 but to-do app is still the "note", it has only changed from 16.3.

File: content/applications/productivity/to_do.rst (hunk 3 above)
Comment (Donapi) [reply content/applications/productivity/to_do.rst:15]: I think it doesn't bring added value to link this page as it doesn't add any further info

File: content/applications/productivity/to_do.rst (hunk 2 above)
Comment (Donapi) [reply content/applications/productivity/to_do.rst:14]: no it's not necessary as we don't show something on the screen

File: content/applications/productivity/to_do.rst (hunk 6 above)
Comment (Donapi) [reply content/applications/productivity/to_do.rst:89]: updated a bit differently

**Comment by Donapi** [comment]
New PR created to target 16.3 
https://github.com/odoo/documentation/pull/5595

**Pull request closed without merging**